.env.template

```diff
@@ -9,8 +11,8 @@
 # WALGREENS_X_XSRF_TOKEN=
 # WALGREENS_XSRF_TOKEN_COOKIE=
 
-CVS_ALLOW_LIST={"MO": ["SAINT LOUIS"], "KS": []}
-CVS_BLOCK_LIST={"MO": [], "KS": []}
+CVS_ALLOW_LIST={"MO": ["SAINT LOUIS"], "KS": [], "IA": []}
+CVS_BLOCK_LIST={"MO": [], "KS": [], "IA": []}
```

Comment:
I'd like to leave additional states out of the template, people can add it to their own .env file.

Reply:
change implemented.